### PR TITLE
chore(migration): update readme and other release docs

### PR DESCRIPTION
**Pull request by kmova**
Signed-off-by: kmova <kiran.mova@mayadata.io>

## Pull Request template

**Why is this PR required? What issue does it fix?**:

This PR updates CI scripts and documents inline with the migration of the code to the new repo. 

**What this PR does?**:
- Updates test scripts
- Update Release process-related docs 
- Update Readme with community links 

**Does this PR require any upgrade changes?**:
No

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -149,3 +150,13 @@ format:
...
 
+# -composite: avoid "literal copies lock value from fakePtr"
+.PHONY: vet
+vet:
+	@echo "--> Running go vet"
+	@go list ./... | grep -v "./vendor/*" | xargs go vet -composites
+
+.PHONY: verify-src
+verify-src: 
+	@echo "--> Checking for git changes post running tests";
```
Comment (akhilerm) [inline GNUmakefile:161]: Why is this required? Checking source files after running tests.

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ci
+
+on:
+  pull_request:
+    branches:
+      # on pull requests to develop and release branches
+      - develop
```
Comment (akhilerm) [inline .github/workflows/pull_request.yml:21]: are we going to use develop or will be switching to main? because once we start releasing, IMO main should be used.

File: .travis.yml [hunk 3]
```diff
@@ -41,15 +41,19 @@ addons:
   apt:
     update: true
 
-before_script:
+install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
   - sudo apt-get install --yes -qq gcc jq
   - sudo apt-get install --yes -qq libudev-dev
   - sudo apt-get install --yes -qq zip
```
Comment (akhilerm) [inline .travis.yml]: Why do local PV provisioner require these dependencies?

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+v2.3.0 / Developing
```
Comment (akhilerm) [inline CHANGELOG.md]: Since we plan to have this released for 2.3, this can be removed.

File: README.md [hunk 5]
```diff
@@ -44,9 +48,20 @@ Please refer to our documentation at [OpenEBS Documentation](http://docs.openebs
...
+
+- [Kubernetes Slack](http://slack.k8s.io/) channels: 
+      * [#openebs](https://kubernetes.slack.com/messages/openebs/)
+      * [#openebs-dev](https://kubernetes.slack.com/messages/openebs-dev/)
+- [Mailing List](https://lists.cncf.io/g/cncf-openebs-users)
+
+### Code of conduct
 
-See the [OpenEBS Community page](https://github.com/openebs/openebs/tree/master/community) for reaching out to the OpenEBS Developers.
+Participation in the OpenEBS community is governed by the [CNCF Code of Conduct](CODE-OF-CONDUCT.md).
```
Comment (akhilerm) [inline README.md:64]: question: Is this a new change that we need to take up in all other repos also? In other repos we have a CODE_OF_CONDUCT.md file present.

File: RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: v1.9.0-RC1 is a GitHub release tag for the release build. v1.9.0 is the release tag that is created after the release criteria are satisfied by the release candidate builds.)
+
+Once the release is triggered, Travis build process has to be monitored. Once Travis build is passed images are pushed to docker hub and quay.io. Images can be verified by going through docker hub and quay.io. Also the images shouldn't have any high-level vulnerabilities.
+
+Images for the different components are published at the following location:
+
+- Dynamic LocalPV Provisioner <br />
+    https://quay.io/repository/openebs/provisioner-localpv?tab=tags <br />
+    https://quay.io/repository/openebs/provisioner-localpv-arm64?tab=tags <br />
+    https://quay.io/repository/openebs/provisioner-localpv-ppc64le?tab=tags <br />
```
Comment (akhilerm) [inline RELEASE.md:36]:
```suggestion
```
This can be removed.

File: RELEASE.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    https://hub.docker.com/r/openebs/provisioner-localpv/tags <br />
+    https://hub.docker.com/r/openebs/provisioner-localpv-arm64/tags <br />
+    https://hub.docker.com/r/openebs/provisioner-localpv-ppc64le/tags <br />
```
Comment (akhilerm) [inline RELEASE.md:39]:
```suggestion
```
This can also be removed.

File: buildscripts/test.sh [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+if [ -z "$ARCH" ]; then
+  echo "platform not specified for running tests. Exiting."
+  exit 1
+fi
+
+# currently tests are run only for amd64
+if [ "$ARCH" != "amd64" ]; then
+  exit 0
+fi
```
Comment (akhilerm) [inline buildscripts/test.sh]: IMO, its safe to remove this. Since the localPV tests does not have dependency on the host.

**Review by akhilerm — CHANGES_REQUESTED** [review]
@kmova Given some comments.

File: GNUmakefile (hunk 1 above)
Comment (kmova) [reply GNUmakefile:161]: to verify if the make fmt has updated any source files. Similarly, if there is some auto gen or go deps update, we should do that as part of PR instead of letting CI silently update and build a code that doesn't match the github branch.

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (kmova) [reply .github/workflows/pull_request.yml:21]: we will never "release" from main -- it is going to be a development branch. Actual releases will be done from tagged branches, hence this convention.

File: .travis.yml (hunk 3 above)
Comment (kmova) [reply .travis.yml]: only zip is required for building. Will remove the rest.

File: RELEASE.md (hunk 6 above)
Comment (kmova) [reply RELEASE.md:36]: we can remove this once the multi-arch builds are available.

File: README.md (hunk 5 above)
Comment (kmova) [reply README.md:64]: yes. it should be like this. https://github.com/openebs/linux-utils/blob/master/CODE_OF_CONDUCT.md

**Review by shubham14bajpai — APPROVED** [review]
lgtm

**Review by akhilerm — APPROVED** [review]
/lgtm
